packages/transactions/src/coders/transaction.test.ts

```diff
@@ -15,27 +16,41 @@ describe('TransactionCoder', () => {
       maturity: 0,
       scriptLength: 0,
       scriptDataLength: 0,
-      inputsCount: 0,
+      inputsCount: 1,
```

Comment:
Only to enforce better testing can we add a different value for each field of the tx on this test?

Reply:
sure, let me add it.  I'll also add additional tests to test input, output and witnesses in isolation

Reply:
done! also updated the `transaction.create` tests